app/models/medium.rb

```diff
@@ -217,8 +201,7 @@ def self.select_quizzables
   end
 
   def self.select_importables
-    Medium.sort_localized.except("RandomQuiz", "Question", "Remark",
-                                 "Manuscript").map { |k, v| [v, k] }
+    Medium.sort_localized.except("RandomQuiz", "Question", "Remark").map { |k, v| [v, k] }
   end
```

Comment:
Why were `Manuscript`s removed here?

Reply:
I do not recall. I vaguely remember that I thought about something in this context, but I do not recall what exactly. Also, it would be out of scope for this PR. So I will reintroduce Manuscripts here.

Reply:
Ah, now I recall. There was no sort "Manuscript" before, so having "Manuscript" within except did not do anything anyway. So it makes sense to remove it and in contrast to what I wrote before, I will leave the code as it is.